README.md

```diff
@@ -1,7 +1,30 @@
 # Help Others
 
-To start, run npm install.
+### Start 
+1. To start, run `npm install` on the backend folder and again on the frontend folder.
+  ```
+  ; cd backend
+  ; npm install
+  ; cd frontend
+  ; npm install
+  ```
+## To run the application 
+2. Start by running the backend server `npm run dev`
+  ```
+  ; cd backend
+  ; npm run dev
+  ```
+3. Then cd into the frontend folder and run `npm start`
+  ```
+  ; cd frontend
+  ; npm run start
+  ```
+## Testing
+#### The Backend (API)
+**Note Environment variables (e.g. JWT secret) are set in the .env. file**
 
-To run the backend cd to the backend folder and then run: npm run dev
-
-For the frontend cd to the frontend folder and then run: npm start
+4. To start the server in test mode (so that it connects to the test DB)
+  ```
+  ; cd backend
+  ; npm run test
```

Comment:
When you demo'd you just did 'npm test' - does this make a diff?

Reply:
Crap you're right, it's npm test! Amending it now